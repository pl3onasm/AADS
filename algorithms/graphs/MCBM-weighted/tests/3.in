MIN 

A1 B1 11
A1 B2 10
A1 B3 2
A1 B4 11
A2 B1 8
A2 B2 5
A2 B4 7
A3 B1 9
A3 B2 2
A3 B3 11
A3 B4 1
A4 B1 7
A4 B3 3
A4 B4 7
