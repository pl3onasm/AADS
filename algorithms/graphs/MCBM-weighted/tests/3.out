
--------------------
 Network
 Vertices: 8
 Edges: 14
--------------------
  A1[4]: B1(1,11), B2(1,10), B3(1,2), B4(1,11)
  A2[3]: B1(1,8), B2(1,5), B4(1,7)
  A3[4]: B1(1,9), B2(1,2), B3(1,11), B4(1,1)
  A4[3]: B1(1,7), B3(1,3), B4(1,7)
  B1[0]
  B2[0]
  B3[0]
  B4[0]
--------------------

--------------------
 Matching
 Cardinality: 4
 Min cost: 15.00
--------------------
  A3 -- B4
  A4 -- B1
  A1 -- B3
  A2 -- B2
--------------------